cmds/bmp/create_baremetals_command_test.go

```diff
@@ -23,7 +24,9 @@ var _ = Describe("create-baremetals command", func() {
 	BeforeEach(func() {
 		args = []string{"bmp", "create-baremetals"}
 		options = cmds.Options{
-			Verbose: false,
+			Verbose:    false,
+			Deployment: "../../test_fixtures/bmp/deployment.yml",
```

Comment:
Hmm, don't really like this. Use the `filepath.Join(...)` function instead to create paths. That's more portable.